configure.py

```diff
@@ -1337,8 +1337,21 @@ def config_info_line(name, help_text):
   """Helper function to print formatted help text for Bazel config options."""
   print('\t--config=%-12s\t# %s' % (name, help_text))
 
+def show_config_info():
+  """Helper function to print bazel command options info"""
+  print('Preconfigured Bazel build configs. You can use any of the below by '
+        'adding "--config=<>" to your build command. See tools/bazel.rc for '
+        'more details.')
+  config_info_line('mkl', 'Build with MKL support.')
+  config_info_line('monolithic', 'Config for mostly static monolithic build.')
+  config_info_line('tensorrt', 'Build with TensorRT support.')
 
 def main():
+
+  if len(sys.argv) == 2 and (sys.argv[1] == '--help' or sys.argv[1] == '-h'):
+    show_config_info()
```

Comment:
Currently this script does much more than print out some of the Bazel config options. I'm not sure if this information, by itself, really makes sense for this script's --help output.

Reply:
Thanks @case540 for the review. I agree the help information here is not that informative enough.  Any suggestion to improve this?

Reply:
One thing you could do is add args corresponding to all the environment variables we use in this script.

Like...
parser.add_argument(
  '--use_jemalloc',
  'Compile using jemalloc as malloc. Alternatively can set environment variable '
  'TF_NEED_JEMALLOC=1 to enable.')

...this would let all of this info also be documented by configure.py --help. I'm not sure how much use this would get though, and we are actively trying to remove things from the configure.py script (so many of these options might be deleted in the near future).